### PR TITLE
data: iAnonymous3000

### PR DESCRIPTION
**Pull request by iAnonymous3000**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/5342"></a>

### CODE REVIEW

File: data/iAnonymous3000.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+  "name": "Sooraj Sathyanarayanan",
+  "type": "personal",
+  "bio": "Security & Privacy Researcher, International Chess Player♟️"
```
Comment (shubhsharma19) [inline data/iAnonymous3000.json]: Please add a comma at the end of your bio

`"bio": "Security & Privacy Researcher, International Chess Player♟️",`

File: data/iAnonymous3000.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+  "avatar": "https://github.com/iAnonymous3000.jpg"
```
Comment (shubhsharma19) [inline data/iAnonymous3000.json]: Same here as well, add `,` at the end of the line number 5

**Review by shubhsharma19 — APPROVED** [review]
LGTM 🎉

**Comment by SurajPratap10** [comment]
Please remove that `avatar`  part @iAnonymous3000  because we don't need that anymore after the update...

**Review by SurajPratap10 — APPROVED** [review]
looks good ! Thanks for your contribution..

**Review by iamthecloverly — APPROVED** [review]

**Review by Arindam200 — APPROVED** [review]

**Review by kumarsonsoff3 — APPROVED** [review]
Looks good, Thanks! 🎉

**Pull request merged**